wiki/Beatmaps/id.md

```diff
@@ -1,78 +1,53 @@
+**Beatmap** merupakan istilah yang merujuk pada level-level permainan yang tersedia untuk dimainkan. Tiap-tiap berkas beatmap mengandung berbagai informasi yang terkait dengan aspek-aspek yang hadir di dalam permainan seperti [*hit object*](/wiki/Hit_Objects), [*timing*](/wiki/Beatmap_Editor/Timing), metadata, pengaturan tingkat kesulitan *(difficulty setting)*, efek-efek visual, dan lain sebagainya.
 
-Beatmap adalah tingkat kesulitan dalam [osu!](/wiki/osu!_glossary), sama seperti tahapan di [Permainan DS](/wiki/Glossary#ds-games). Dalam bentuk paling dasar mereka, beatmap memiliki lagu dan berbagai [hit objects](/wiki/hit_objects) yang ditempatkan sesuai dengan irama musik. Mereka juga dapat memiliki gambar latar belakang, video, bahkan [storyboard](/wiki/storyboard).
+Beatmap-beatmap yang dibuat untuk satu berkas lagu yang sama pada umumnya akan secara otomatis terkelompokkan ke dalam satu **[beatmapset](Beatmapsets)** utuh. Kamu dapat menjumpai beragam jenis beatmapset yang telah [diunggah ke situs osu!](/wiki/Glossary#bss) pada laman [daftar beatmap](https://osu.ppy.sh/beatmapsets), di mana pada masing-masing beatmapset tersebut kamu akan dihadapkan dengan satu atau lebih [tingkat kesulitan](/wiki/Difficulties) di dalamnya.
 
-Beatmap diunggah menggunakan [BSS](/wiki/BSS) (Beatmap Submission System) dan kemudian melalui proses persetujuan yang disebut [modding](/wiki/modding) untuk mempertahankan standar.
+## Status beatmap
 
-Anda dapat menemukan beatmap pada laman [Beatmaps Listing](https://osu.ppy.sh/beatmapsets) osu! web.
+Beatmap-beatmap yang telah diunggah ke situs osu! terbagi ke dalam beberapa kategori status berdasarkan tingkat aktivitas, status nominasi, dan popularitas dari beatmap-beatmap yang bersangkutan sebagai berikut:
 
-## Status Ranking
+- [Graveyard](#graveyard)
+- [Work in Progress / Pending](#work-in-progress-dan-pending)
+- [Qualified](#qualified)
+- [Ranked](#ranked)
+- [Loved](#loved)
 
-Baik beatmap yang berstatus ranked maupun approved memerlukan persetujuan dari dua [Beatmap Nominators](/wiki/Beatmap_Nominators). Mereka dapat mengubah statistik pada profil seorang pemain yang membuat mereka berbeda dari beatmap [pending](#pending) atau [graveyarded](#graveyard).
+### Graveyard
 
-### Ranked
+Graveyard merupakan status beatmap yang disematkan kepada beatmap-beatmap yang telah lama diabaikan oleh pembuatnya. Suatu beatmap yang berstatus [Work in Progress dan Pending](#work-in-progress-and-pending) akan secara otomatis dipindahkan ke dalam Graveyard apabila beatmap yang bersangkutan tidak diperbaharui selama 4 minggu. Beatmap-beatmap yang berstatus Graveyard tidak dilengkapi dengan *leaderboard* dan tidak berpengaruh terhadap statistik permainan yang muncul di halaman profil pemain, namun tetap dapat diunduh dan dimainkan seperti biasa. Adapun beatmap-beatmap ini ditandai dengan simbol tanda tanya (![](/wiki/shared/status/graveyard.png)) yang dapat dilihat pada pojok kiri atas layar Song Selection.
```

Comment:
```suggestion
Graveyard merupakan status beatmap yang disematkan kepada beatmap-beatmap yang telah lama diabaikan oleh pembuatnya. Suatu beatmap yang berstatus [Work in Progress dan Pending](#work-in-progress-dan-pending) akan secara otomatis dipindahkan ke dalam Graveyard apabila beatmap yang bersangkutan tidak diperbaharui selama 4 minggu. Beatmap-beatmap yang berstatus Graveyard tidak dilengkapi dengan *leaderboard* dan tidak berpengaruh terhadap statistik permainan yang muncul di halaman profil pemain, namun tetap dapat diunduh dan dimainkan seperti biasa. Adapun beatmap-beatmap ini ditandai dengan simbol tanda tanya (![](/wiki/shared/status/graveyard.png)) yang dapat dilihat pada pojok kiri atas layar Song Selection.
```

Reply:
oh, this is intended actually because the sub-heading it is referring to (Line 25) has also been translated into ```### Work in Progress dan Pending```. keeping it as is with the original (```#work-in-progress-dan-pending```) will actually break the hyperlink :(

Reply:
yeah, he's referring to the first occurrence you haven't adjusted

Reply:
oh i see lol, yeah mb